Source/RunActivity/Viewer3D/RollingStock/MSTSLocomotiveViewer.cs

```diff
@@ -2147,7 +2147,7 @@ public virtual int GetDrawIndex()
                         {
                             if (Locomotive.CruiseControl != null)
                             {
-                                if ((Locomotive.CruiseControl.SpeedRegMode == Simulation.RollingStocks.SubSystems.CruiseControl.SpeedRegulatorMode.Auto && !Locomotive.CruiseControl.DynamicBrakePriority) || Locomotive.DynamicBrakeIntervention > 0)
+                                if (Locomotive.CruiseControl.SpeedRegMode == Simulation.RollingStocks.SubSystems.CruiseControl.SpeedRegulatorMode.Auto && !Locomotive.CruiseControl.DynamicBrakePriority)
```

Comment:
Hi Phillip,

Wondering why is there a change in the logic here. Surely this has nothing to do with units of measure?

If it's a bug-fix, please move it to a separate PR.

Thanks,

Chris

Reply:
Noticed while testing cab views that anything with cruise control would not show the dynamic brake setting at all due to a change in dynamic brake code made after cruise control was added (basically, dynamic brake intervention used to always be 0 unless cruise control was on, now dynamic brake intervention will change from 0 whenever the dynamic brake is used).

This can be done as a separate mini PR, got a bit I want to change with the units anyway as some more feedback has pointed out how non-linear gauges are effectively impossible to calibrate-an option for exponential units will be needed.